src/main/java/seedu/address/logic/commands/contact/CDeleteCommand.java

```diff
@@ -17,39 +18,61 @@
  */
 public class CDeleteCommand extends Command {
 
-    public static final String COMMAND_WORD = "delete";
+    public static final String COMMAND_WORD = "cdelete";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the contact identified by the index number used in the displayed contact list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + "Parameters: INDEX[-INDEX] (must be a positive integer)\n"
+            + "Example 1: " + COMMAND_WORD + " 1\n"
+            + "Example 2: " + COMMAND_WORD + " 2-5";
 
     public static final String MESSAGE_DELETE_CONTACT_SUCCESS = "Deleted Contact: %1$s";
 
-    private final Index targetIndex;
+    private final Range targetRange;
 
     public CDeleteCommand(Index targetIndex) {
```

Comment:
Do we still need to keep this constructor? since you have already made single index into a range from itself to itself before passing it into CDeleteCommand

Reply:
I kept this constructor so that the test cases can still pass for the CI checks for merging, since some test cases use this constructor. Will remove after updating the test cases to use Range instead.